### PR TITLE
Docs folder standardization and other fixes

### PR DESCRIPTION
**Pull request by luizcmarin**

### CODE REVIEW

File: templates/_call-stack-item.php [hunk 1]
```diff
@@ -11,27 +11,26 @@
 /* @var $isVendorFile bool */
 /* @var $this \Yiisoft\ErrorHandler\Renderer\HtmlRenderer */
 $icon = <<<HTML
-<svg class="external-link" width="20" height="20" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+<svg class="external-link" width="20" height="20" viewBox="0 0 24 24" xmlns="https://www.w3.org/2000/svg">
```
Comment (samdark) [inline templates/_call-stack-item.php:14]:
```suggestion
<svg class="external-link" width="20" height="20" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```

File: templates/_call-stack-item.php [hunk 2]
```diff
@@ -11,27 +11,26 @@
...
 </svg>
 HTML;
 ?>
-<li class="<?= (!empty($lines) && ($index === 1 || !$isVendorFile)) ? 'application ' : '' ?>call-stack-item"
-    data-line="<?= (int) ($line - $begin) ?>">
+<li class="<?= (!empty($lines) && ($index === 1 || !$isVendorFile)) ? 'application ' : '' ?>call-stack-item" data-line="<?= (int) ($line - $begin) ?>">
     <div class="element-wrap">
         <div class="flex-1">
-            <?php if ($file !== null): ?>
+            <?php if ($file !== null) : ?>
```
Comment (samdark) [inline templates/_call-stack-item.php:24]:
```suggestion
            <?php if ($file !== null): ?>
```

File: templates/_call-stack-item.php [hunk 3]
```diff
@@ -11,27 +11,26 @@
...
                 <span class="file-name">
                     <?= "{$index}. in {$this->htmlEncode($file)}" ?>
-                    <?php if ($this->traceHeaderLine !== null): ?>
+                    <?php if ($this->traceHeaderLine !== null) : ?>
```
Comment (samdark) [inline templates/_call-stack-item.php:27]:
```suggestion
                    <?php if ($this->traceHeaderLine !== null): ?>
```

File: templates/_call-stack-item.php [hunk 4]
```diff
@@ -11,27 +11,26 @@
...
                         <?= strtr($this->traceHeaderLine, ['{file}' => $file, '{line}' => $line + 1, '{icon}' => $icon]) ?>
                     <?php endif ?>
                 </span>
             <?php endif ?>
 
-            <?php if ($function !== null): ?>
+            <?php if ($function !== null) : ?>
```
Comment (samdark) [inline templates/_call-stack-item.php:33]:
```suggestion
            <?php if ($function !== null): ?>
```

File: templates/_call-stack-item.php [hunk 5]
```diff
@@ -40,25 +39,25 @@
             <?php endif ?>
         </div>
 
-        <?php if ($line !== null): ?>
+        <?php if ($line !== null) : ?>
```
Comment (samdark) [inline templates/_call-stack-item.php:42]:
```suggestion
        <?php if ($line !== null): ?>
```

File: templates/_call-stack-item.php [hunk 6]
```diff
@@ -40,25 +39,25 @@
...
             <div><?= sprintf('at line %d', $line + 1) ?></div>
         <?php endif ?>
     </div>
-    <?php if (!empty($lines)): ?>
+    <?php if (!empty($lines)) : ?>
```
Comment (samdark) [inline templates/_call-stack-item.php:46]:
```suggestion
    <?php if (!empty($lines)): ?>
```

File: templates/_call-stack-item.php [hunk 7]
```diff
@@ -40,25 +39,25 @@
...
         <div class="element-code-wrap">
             <div class="code-wrap">
                 <div class="error-line"></div>
-                <?php for ($i = $begin; $i <= $end; ++$i): ?><div class="hover-line"></div><?php endfor ?>
+                <?php for ($i = $begin; $i <= $end; ++$i) : ?><div class="hover-line"></div><?php endfor ?>
```
Comment (samdark) [inline templates/_call-stack-item.php:50]:
```suggestion
                <?php for ($i = $begin; $i <= $end; ++$i): ?><div class="hover-line"></div><?php endfor ?>
```

File: templates/_call-stack-item.php [hunk 8]
```diff
@@ -40,25 +39,25 @@
...
                 <div class="code">
-                    <?php for ($i = $begin; $i <= $end; ++$i): ?><span class="lines-item"><?= (int) ($i + 1) ?></span><?php endfor ?>
+                    <?php for ($i = $begin; $i <= $end; ++$i) : ?><span class="lines-item"><?= (int) ($i + 1) ?></span><?php endfor ?>
```
Comment (samdark) [inline templates/_call-stack-item.php:52]:
```suggestion
                    <?php for ($i = $begin; $i <= $end; ++$i): ?><span class="lines-item"><?= (int) ($i + 1) ?></span><?php endfor ?>
```

File: templates/_call-stack-item.php [hunk 9]
```diff
@@ -40,25 +39,25 @@
...
                     <pre class="codeBlock language-php"><?php
-                        // Fill empty lines with a whitespace to avoid rendering problems in Opera.
-                        for ($i = $begin; $i <= $end; ++$i) {
-                            echo (trim($lines[$i]) === '') ? " \n" : $this->htmlEncode($lines[$i]);
-                        }
-                        ?></pre>
+                                                        // Fill empty lines with a whitespace to avoid rendering problems in Opera.
```
Comment (samdark) [inline templates/_call-stack-item.php:54]: Why adding that many spaces?

File: templates/development.php [hunk 10]
```diff
@@ -37,307 +36,310 @@
...
-                <span><?= $this->htmlEncode($throwable->getName())?></span>
-                &mdash;
-                <?= $exceptionClass ?>
-            <?php else: ?>
-                <span><?= $exceptionClass ?></span>
-            <?php endif ?>
+    <header>
+        <div class="tools">
+            <a href="#" title="Dark Mode" id="dark-mode">
+                <svg width="28" height="28" fill="none" xmlns="https://www.w3.org/2000/svg">
```
Comment (samdark) [inline templates/development.php:42]:
```suggestion
                <svg width="28" height="28" fill="none" xmlns="http://www.w3.org/2000/svg">
```

File: templates/development.php [hunk 11]
```diff
@@ -37,307 +36,310 @@
...
+                    <path fill-rule="evenodd" clip-rule="evenodd" d="M27.0597 19.06c-2.5312.9821-5.2934 1.2069-7.9501.6472-2.6568-.5598-5.0934-1.8799-7.0133-3.7998-1.9198-1.9198-3.24001-4.3565-3.79975-7.01324-.55974-2.65674-.33489-5.41892.64718-7.95016-3.04045 1.1849-5.57175 3.39424-7.15685 6.24657C.201778 10.0429-.337628 13.3592.26179 16.5668c.599419 3.2077 2.30004 6.1053 4.80824 8.1928C7.57822 26.847 10.7366 27.9931 13.9997 28c2.8246.0007 5.5833-.8527 7.9141-2.4481 2.3307-1.5955 4.1245-3.8585 5.1459-6.4919z" fill="#787878" />
+                </svg>
+            </a>
+
+            <a href="#" title="Light Mode" id="light-mode">
+                <svg width="32" height="32" fill="none" xmlns="https://www.w3.org/2000/svg">
```
Comment (samdark) [inline templates/development.php:48]:
```suggestion
                <svg width="32" height="32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

File: templates/development.php [hunk 12]
```diff
@@ -37,307 +36,310 @@
...
+                    <path d="M15 2h2v5h-2V2zM21.6875 8.9l3.506-3.50699 1.414 1.415-3.506 3.50599-1.414-1.414zM25 15h5v2h-5v-2zM21.6875 23.1l1.414-1.414 3.506 3.506-1.414 1.415-3.506-3.507zM15 25h2v5h-2v-5zM5.39453 25.192l3.506-3.506 1.41397 1.415-3.50597 3.506-1.414-1.415zM2 15h5v2H2v-2zM5.39453 6.80801l1.414-1.415L10.3145 8.9l-1.41397 1.414-3.506-3.50599zM16 10c-1.1867 0-2.3467.3519-3.3334 1.0112-.9867.6593-1.7557 1.5963-2.2099 2.6927-.4541 1.0964-.57292 2.3028-.3414 3.4666.2315 1.1639.8029 2.233 1.6421 3.0721.8391.8392 1.9082 1.4106 3.0721 1.6421 1.1638.2315 2.3702.1127 3.4666-.3414 1.0964-.4541 2.0334-1.2232 2.6927-2.2099C21.6481 18.3467 22 17.1867 22 16c0-1.5913-.6321-3.1174-1.7574-4.2426C19.1174 10.6321 17.5913 10 16 10z" fill="#989898" />
+                </svg>
+            </a>
+
+            <a href="https://stackoverflow.com/search?<?= http_build_query(['q' => $exceptionMessage]) ?>" title="Search error on Stackoverflow" target="_blank">
+                <svg width="28" height="32" fill="none" xmlns="https://www.w3.org/2000/svg">
```
Comment (samdark) [inline templates/development.php:54]:
```suggestion
                <svg width="28" height="32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

File: templates/development.php [hunk 13]
```diff
@@ -37,307 +36,310 @@
...
+                    <path d="M23.312 29.151v-8.536h2.849V32H.458008V20.615H3.29701v8.536H23.312zM6.14501 26.307H20.469v-2.848H6.14501v2.848zm.35-6.468L20.47 22.755l.599-2.76-13.96899-2.912-.605 2.756zm1.812-6.74L21.246 19.136l1.203-2.6-12.93699-6.041-1.204 2.584-.001.02zm3.61999-6.38L22.88 15.86l1.813-2.163L13.74 4.562l-1.803 2.151-.01.006zM19 0l-2.328 1.724 8.541 11.473 2.328-1.724L19 0z" fill="#787878" />
+                </svg>
+            </a>
+            <a href="https://www.google.com/search?<?= http_build_query(['q' => $exceptionMessage]) ?>" title="Search error on Google" target="_blank">
+                <svg width="24" height="24" fill="none" xmlns="https://www.w3.org/2000/svg">
```
Comment (samdark) [inline templates/development.php:59]:
```suggestion
                <svg width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

File: templates/development.php [hunk 14]
```diff
@@ -37,307 +36,310 @@
...
-            $originalException === $throwable && $originalException instanceof ErrorException
-                ? $originalException->getBacktrace()
-                : $throwable->getTrace()
-        ) ?>
-    </div>
-    <?php if ($request && ($requestInfo = $this->renderRequest($request)) !== ''): ?>
-        <div class="request">
-            <?= $requestInfo ?>
+            <a href="#" id="copy-stacktrace" title="Copy the stacktrace for use in a bug report or pastebin">
+                <svg width="26" height="30" fill="none" xmlns="https://www.w3.org/2000/svg">
```
Comment (samdark) [inline templates/development.php:91]:
```suggestion
                <svg width="26" height="30" fill="none" xmlns="http://www.w3.org/2000/svg">
```

File: templates/development.php [hunk 15]
```diff
@@ -37,307 +36,310 @@
...
+                    </p>
+                <?php endif ?>
+                <p>
+                    <a href="https://www.yiiframework.com/" target="_blank" rel="noopener noreferrer">Yii Framework</a>
+                    /
+                    <a href="https://github.com/yiisoft/docs/blob/master/guide/en/runtime/handling-errors.md" target="_blank" rel="noopener noreferrer">Error Handling Guide</a>
+                </p>
+            </div>
+
+            <svg width="256" height="224" fill="none" xmlns="https://www.w3.org/2000/svg">
```
Comment (samdark) [inline templates/development.php:134]:
```suggestion
            <svg width="256" height="224" fill="none" xmlns="http://www.w3.org/2000/svg">
```

**Review by samdark — CHANGES_REQUESTED** [review]

File: templates/_call-stack-item.php (hunk 9 above)
Comment (luizcmarin) [reply templates/_call-stack-item.php:54]: vscode..... I'm innocent

**Review by samdark — APPROVED** [review]

**Pull request merged**